portal-impl/src/com/liferay/portlet/messageboards/service/impl/MBMessageLocalServiceImpl.java

```diff
@@ -1979,11 +1979,13 @@ protected String getDiscussionMessageSubject(String subject, String body)
 
 		subject = HtmlUtil.extractText(body);
 
-		if (subject.length() <= 50) {
-			return subject;
+		int pos = Math.min(subject.length(), 50);
```

Comment:
The Math.min method is not really needed we can also use the old code. 
But I found it more readable with the pos variable. 
But you are right the method is not needed at all.
I think that the value 50 need to be a static final int class variable. Because it is a fixed number. Are you with me on that?